Add translations for settings modal and some missing pl

miqs1992 opened the pull request:
## Description

<!-- Explains the goal of this PR -->

- Moved some hardcoded texts into translations in SettingsModal 
- Use old translation values in LayersModal (I don't want to add new values if there are old ones present)
- Added some missing Polish translations

### Preview

<!-- Please add screenshots and/or gif that shows visual changes (if applicable) -->

SettingsModal:
<img width="349" height="648" alt="Screenshot 2025-07-25 at 14 55 39" src="https://github.com/user-attachments/assets/b9620812-bc82-490f-b787-6ccc6251e1c1" />

LayersModal:
<img width="357" height="181" alt="Screenshot 2025-07-25 at 14 55 23" src="https://github.com/user-attachments/assets/b65324d0-3e67-4542-9f5d-63a0e4ffd871" />

Time window selector:
<img width="538" height="347" alt="Screenshot 2025-07-25 at 09 31 46" src="https://github.com/user-attachments/assets/e42bbf2e-ec25-4398-af68-9fa938d80fa2" />
<img width="412" height="111" alt="Screenshot 2025-07-25 at 09 31 37" src="https://github.com/user-attachments/assets/ed690cc3-aea0-4189-a8b3-3143a3298fd5" />

### Double check

- [ ] I have tested my parser changes locally with `poetry run test_parser "zone_key"`
- [X] I have run `pnpx prettier@2 --write .` and `poetry run format` in the top level directory to format my changes.

VIKTORVAV99 reviewed (approved):
LGTM! 🎉

VIKTORVAV99 commented:
@miqs1992 can you allow edits by maintainers please so I can merge master into this?

miqs1992 commented:
@VIKTORVAV99 I'm not sure if I can, as I forked this to an organisation, but I will confirm that. In the meantime, I added you manually and rebased the branch
